pkg/cmd/chat/push/push.go

```diff
@@ -0,0 +1,155 @@
+package push
+
+import (
+	"encoding/json"
+
+	stream_chat "github.com/GetStream/stream-chat-go/v5"
+	"github.com/GetStream/stream-cli/pkg/config"
+	"github.com/GetStream/stream-cli/pkg/utils"
+	"github.com/MakeNowJust/heredoc"
+	"github.com/spf13/cobra"
+)
+
+func NewCmds() []*cobra.Command {
+	return []*cobra.Command{
+		updateCmd(),
+		testCmd(),
+	}
+}
+
+func updateCmd() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "upsert-pushprovider --properties [raw-json]",
+		Short: "Create or updates a push provider",
+		Long: `
+			The "--properties" parameter expects a raw json string that can be
+			unmarshalled into a stream_chat.PushProvider object on the Go SDK side.
+			See the example section.
+			Available properties:
+			type
+			name
+			description
+			disabled_at
+			disabled_reason
+					
+			apn_auth_key
+			apn_key_id
+			apn_team_id
+			apn_topic
+					
+			firebase_credentials
```

Comment:
For v2 we have:
- firebase_notification_template
- firebase_apn_template

Reply:
I'll need this PR to be merged first: https://github.com/GetStream/stream-chat-go/pull/218

Reply:
added these 3

Reply:
we can still provide `firebase_apn_template` and `firebase_notification_template`. why did you drop them?

Reply:
from where? it's there